service/share/testing.go

```diff
@@ -138,3 +177,7 @@ func NewBrokenAvailability() Availability {
 func (b *brokenAvailability) SharesAvailable(context.Context, *Root) error {
 	return ErrNotAvailable
 }
+
+func (b *brokenAvailability) StartPeerDiscovery() {}
+
+func (b *brokenAvailability) StopPeerDiscovery() {}
```

Comment:
Can we add more comments/functionality or remove these?

Reply:
We need these method so brokenAvailability can implement `Availability` interface.

Reply:
Then please leave a comment for them. Otherwise, this can be removed by accident later 🙏

Reply:
removed at all